src/sentry/migrations/0608_add_billing_email_verified.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.23 on 2023-11-28 21:54
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0607_drop_externalactor_actorid"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="useremail",
+            name="is_billing_notification_verified",
```

Comment:
Is it possible to rename this, and/or make this feature general purpose (i.e. not only used for billing)?
We usually avoid exposing billing and usage concepts in `sentry`.

Reply:
Hmm, that's a good point.  The other way to model this is with a separate model that only exists in `getsentry` that relates back to `UserEmail`.

Reply:
A separate table could be useful as you could also store the time of the most recent 'verification' for the email receipt and would also address having the billing role present in sentry.